resources/public/api/definition.json

```diff
@@ -18,7 +18,18 @@
         "endpointsEnabled": true,
         "access": {
           "type": "PUBLIC"
-        }
+        },
+        "fieldDefinitions": [{
+          "name": "notificationUrl",
+          "description": "What is your notification web address for us to send push notifications to?",
+          "type": "URL",
```

Comment:
This should be a PPNSField

Reply:
Again see e.g. https://github.com/hmrc/goods-movement-system-haulier-api/blob/master/app/uk/gov/hmrc/goodsmovementsystemhaulierapi/platform/views/definition.scala.txt#L30